### DISCUSSION TITLE
How to set max value for Radial Bar?

### DISCUSSION THREAD

**Discussion by deebov** [recharts/recharts#5024] [Q&A]
Hey! I'm trying to reproduce a radial chart like in the screenshot. As you can see, not all circles are filled. However, I realized that recharts takes the value of a circle with the highest value as the maximum for the entire radial chart. And based on that number, it renders the other circles relative to that.

Is there a way to set a max value for the radial chart?

**What I'm trying to achieve:**

<img width="391" alt="image" src="https://github.com/user-attachments/assets/6314ab0c-1c62-49a3-be01-d3add615a7c9">

**What I have:**

<img width="460" alt="image" src="https://github.com/user-attachments/assets/16ec3f89-d3b2-4435-9e31-6153402b3396">

**Comment by ckifer** [comment]
Hey there, this should work
https://codesandbox.io/p/sandbox/simple-radial-bar-chart-gnwjjg

You'll have to add a `PolarAngleAxis`
```tsx
      <PolarAngleAxis
        tick={false}
        domain={[0, 100]}
        type="number"
        // @ts-expect-error this prop works but isn't correct in TS types at the moment
        reversed
      />
```

**Comment by machour** [comment]
Tried today, this doesn't seem to work.

I need my circles to go from 0 to 10, I even generated a fake (green) entry with a value of 10 to set the max, but values of 8 are also rendered as full circles:

<img width="442" alt="Screenshot 2025-05-22 at 10 28 02 AM" src="https://github.com/user-attachments/assets/7bab72a6-fb71-4888-98d6-748dd14b366d" />

<img width="525" alt="Screenshot 2025-05-22 at 10 28 09 AM" src="https://github.com/user-attachments/assets/9f8ba5ff-96bb-4859-80dd-ef224f05783d" />

**Reply by ckifer** [reply]
Seems to work here as I mentioned above https://codesandbox.io/p/devbox/simple-radial-bar-chart-forked-8z86c8

`domain` being the key piece

Can you make a codesandbox reproduction?
